src/server/api/routers/traces.ts

```diff
@@ -360,4 +360,51 @@ export const traceRouter = createTRPCRouter({
         throw e;
       }
     }),
+  deleteMany: protectedProjectProcedure
```

Comment:
Great to do this in one call. Let's only have one deleteMany procedure and remove the old delete [single] with the goal of keeping this maintainable. Need to update the current delete button to use the delete many and pass a list with the single trace to be deleted